tests/nlu/featurizers/test_lm_featurizer.py

```diff
@@ -737,7 +437,7 @@ def test_log_deprecation_warning_with_old_config(text: str, caplog: LogCaptureFi
     assert "deprecated component HFTransformersNLP" in caplog.text
 
 
-@pytest.mark.skip(reason="Results in random crashing of github action workers")
+@skip_on_CI
 def test_preserve_sentence_and_sequence_features_old_config():
     attribute = "text"
     message = Message.build("hi there")
```

Comment:
Not sure, but it could be these kind of tests where multiple models are loaded in the same test which is breaking the CI.

Reply:
Ahh that's not a bad thought. I will try and figure that out